interfaces/builtin/modem_manager.go

```diff
@@ -1218,12 +1221,28 @@ LABEL="mm_zte_port_types_end"
 # that don't have this tag.  MM will still get the udev 'add' event for the
 # device a short while later and then process it as normal.
 
-ACTION!="add|change|move", GOTO="mm_candidate_end"
+ACTION!="add|change|move|bind", GOTO="mm_candidate_end"
```

Comment:
do these changes come from an update to the modem-manager deb?

Reply:
Yes, the udev rules changes come from the latest MM debian package.